src/contexts/layout-context.tsx

```diff
@@ -89,9 +89,9 @@ export const LayoutProvider: React.FC<PropsWithChildren<{ pageContext: PageConte
     if (activePageData?.page.languages) {
       languages = activePageData.page.languages; // Use language overrides from the nav data first if possible
     } else if (pageContext?.languages) {
-      languages = pageContext.languages.map(stripSdkType) as LanguageKey[]; // Use pageContext languages if available, this is generated for MDX pages
+      languages = Array.from(new Set(pageContext.languages.map(stripSdkType))) as LanguageKey[]; // Use pageContext languages if available, this is generated for MDX pages
```

Comment:
Solves a separate problem where langs could be duplicated when there's a full bevy of realtime and rest langs